docs/tutorial/t-get-signed-certificates.md

```diff
@@ -0,0 +1,25 @@
+# Get Signed Certificates
```

Comment:
I'm not sure this section is useful. There's an actions tab in Charmhub (https://charmhub.io/self-signed-certificates/actions?channel=edge)

Reply:
The section is removed.